app/views/devise/passwords/new.html.erb

```diff
@@ -11,7 +11,6 @@
 <%= simple_form_for(
       @password_reset_email_form,
       url: user_password_path,
-      html: { autocomplete: 'off', method: :post },
```

Comment:
`method: :post` is default, so removed it.

Reply:
> `method: :post` is default, so removed it.

I don't think it's actually so straight-forward, there may be cases where we want explicit `method: :post`. Unclear which of these are candidates. Could be safer to back out the changes.

https://github.com/rails/rails/blob/6f0d1ad14b92b9f5906e44740fce8b4f1c7075dc/actionview/lib/action_view/helpers/form_helper.rb#L1595

Reply:
Added them back in 7dead6b for `simple_form_for` where the record is a persisted model (e.g. `current_user`).